README.md

```diff
@@ -43,19 +43,18 @@ You will be prompted to install the Python dependencies during the first use.
 - _**[Python 3.7.4](https://www.python.org/downloads/)**_: Make sure you've added python and pip to your PATH in your environment variables. (1)
 - _**[Python VS Code extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python)**_: This will be installed automatically from the marketplace when you install Device Simulator Express.
 
-The following dependecies can be installed for you by the extension by clicking yes when you are prompted to (**except** `pywin32` which is needed only on Windows platform). (2)
-
-- _**Playsound**_  
-  install by typing the following commands in a console: `pip install playsound`
-
-- _**Pywin 32**_  
-  install by typing the following commands in a console (only for Windows computers, you must run it manually): `pip install pywin32`
-- _**Python-Socketio**_  
-   install by typing the following commands in a console: `pip install python-socketio`
-- _**Requests**_  
-   install by typing the following commands in a console: `pip install requests`
-- _**Application Insights**_  
-  install by typing the following commands in a console: `pip install applicationinsights`
+- Python Modules for Deploy to Device

```

Comment:
This is not on the scope of this PR, so don't worry! I wonder if we should move these install instructions from the `README.md` and just point to the `install.md` instead? Otherwise we just have dupe information (or we can make the whole README overwhelming for new users).

Reply:
I think that the instructions were here beforehand too (or at least the pip install commands). But yeah, it would maybe be a good idea to move all installation-related stuff to install.md for clarity (I noticed that lots of parts repeat between docs)

Reply:
Yup -- you just followed the pattern of what we have before, so it's all good! However, I think it might be worth adding a work item to do that in the future? (basically linking stuff at `README `so it's more doc-based).